test/coverage-test/fixtures/configs/vitest.config.workspace.ts

```diff
@@ -0,0 +1,20 @@
+import { defineConfig } from "vitest/config";
+
+export default defineConfig({
+  test: {
+    projects: [
+      {
+        test: {
+          name: "project1",
+          root: "fixtures/workspaces/project/project1",
+        },
+      },
+      {
+        test: {
+          name: "project2",
+          root: "fixtures/workspaces/project/project2",
+        },
+      },
+    ]
+  }
+});
```

Comment:
Let's add a `project-shared` that contains `project-shared/src/utils.ts`, that's used by `id.ts` and `konst.ts`. Then make sure that `utils.ts` does not show up in report when project filter is used even though it is covered file.